File: Server/src/commonStructures/networkOwnedObjectsList.ts
/**
 * This class is used to store the list of objects owned by a player.
 */
export default class NetworkOwnedObjectsList {
    /**
     * The id of the player.
     */
    public playerId:number;
    /**
     * The list of objects owned by the player.
     */
    public ownedObjects:number[];
    /**
     * The constructor for the NetworkOwnedObjectsList class.
     * @param playerId The id of the player.
     * @param ownedObjects The list of objects owned by the player.
     */
    constructor(playerId:number,ownedObjects:number[]) {
        this.playerId = playerId;
        this.ownedObjects = ownedObjects;
    }
}

File: Assets/Scripts/Network/CommonStructures/NetworkOwnedObjectsList.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Network
{
    /// <summary>
    /// This class is used to store the list of objects owned by a player.
    /// </summary>
    public class NetworkOwnedObjectsList
    {
        /// <summary>
        /// The id of the player.
        /// </summary>
        public int playerId;
        /// <summary>
        /// The list of objects owned by the player.
        /// </summary>
        public int[] ownedObjects;
    }
}
